Dockerfile.apk

```diff
@@ -13,6 +13,7 @@ COPY kong.apk.tar.gz /tmp/kong.apk.tar.gz
 
 RUN set -ex; \
     apk add bash curl ca-certificates; \
+    apk add --no-cache --virtual .build-deps tar gzip; \
```

Comment:
per description of PR; how would this NOT add curl? it's been added one line up?

Reply:
how very odd ... not sure how I missed that. The curl should be one line lower as `.build-deps` gets removed later in the Dockerfile